.gitignore

```diff
@@ -19,20 +19,21 @@ startup_stm32f10x_md_gcc.s
 docs/Manual.pdf
 README.pdf
 
-# artifacts of top-level Makefile
+# artefacts of top-level Makefile
 /downloads
 /tools
 /build
 # local changes only
 make/local.mk
 
+# artefacts for VisualGDB (running in Visual Studio)
 mcu.mak
 mcu.mak.old
 stm32.mak
 
-# Artefacts for Visual Studio Code
-/.vscode/
+# artefacts for Visual Studio Code
+/.vscode
 
-# Artefacts for CLion
-/cmake-build-debug/
```

Comment:
I assume that user created files that have names identical to directories that IDEs try to create will lead to problems, so they probably shouldn't be ignored silently.